frontend/src/views/pages/vehicleLists/UserVehicleLists.vue

```diff
@@ -0,0 +1,313 @@
+<template>
+  <div>
+    <CCard>
+      <CCardHeader>
+        <slot name="header">Table of custom vehicle lists</slot>
+      </CCardHeader>
+      <CCardBody>
+        <CRow class="d-flex justify-content-between pl-3 pr-3">
+          <CButton color="primary mb-3" @click="addingVehicleList = true">Create a custom vehicle list</CButton>
+          <CRow class="m-0">
+            <CButton color="success mb-3 mr-2" @click="applyAction" v-if="selectedBulkAction != 'Bulk Action'">Apply</CButton>
+            <CButton color="secondary mb-3 mr-2" @click="cancelAction" v-if="selectedBulkAction != 'Bulk Action'">Cancel</CButton>
+            <CSelect 
+              :options="bulkActions"
+              :value.sync="selectedBulkAction"
+              :disabled="selectedBulkAction != 'Bulk Action'"
+            />
```

Comment:
Switched the CSelect to a button for a more intuitive process. Since we didn't really have any other bulk actions, the button does make more sense. You can review the code [here](https://github.com/purplesmurf1998/Autotracks/pull/62/commits/d63f3fba316c455b116d3997247a318df62710db)